frontend/src/component/insights/components/WidgetTitle/WidgetTitle.tsx

```diff
@@ -0,0 +1,32 @@
+import type { FC, ReactNode } from 'react';
+import { Typography } from '@mui/material';
+import { HelpIcon } from 'component/common/HelpIcon/HelpIcon';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import InfoOutlined from '@mui/icons-material/InfoOutlined';
+
+export const WidgetTitle: FC<{
+    title: ReactNode;
+    tooltip?: ReactNode;
+}> = ({ title, tooltip }) => (
+    <>
+        <Typography
+            variant='h3'
+            sx={(theme) => ({
+                marginBottom: theme.spacing(3),
```

Comment:
Why does this component control its own margins? Isn't that better handled by the parent?

Reply:
I noticed where we use it, the parent also sets up a flag gap. Seems like we're doubling up on spacing for some reason?